### PR TITLE
Fix for docker api container DB hosted outside docker environment.

### PR DESCRIPTION
**Pull request by vishalraina**
Following issues have been fixed for API docker containers:
1. Added dbhost property to enable DB hosted outside docker.
2. Fixed documentation link for GitHub private repo.
3.Added example docker environment file.
4. Added documentation on Hygieia wiki on how to run API Docker container with environment file.

### CODE REVIEW

**Comment by vishalraina** [comment]
@rvema Please review the PR and let me know if you have any questions.

**Comment by rvema** [comment]
@vishalraina I am looking into this PR and will merge in a day.

File: api/docker/properties-builder.sh [hunk 1]
```diff
@@ -33,7 +33,12 @@ dbusername=${SPRING_DATA_MONGODB_USERNAME:-dashboarduser}
 #Database Password - default is blank
 dbpassword=${SPRING_DATA_MONGODB_PASSWORD:-dbpassword}
 
-#API encryption key. Optional. See http://capitalone.github.io/Hygieia/setup.html#encryption-for-private-repos	
+#dbhostport
```
Comment (rvema) [inline api/docker/properties-builder.sh]: please remove this duplicate property which is commented out, It will cause confusion.

**Comment by rvema** [comment]
@vishalraina looks good with one minor change, please update the PR.

**Comment by vishalraina** [comment]
@rvema thank you, for your review comments.
I have removed duplicate dbhostport property.
Please review again.

**Comment by rvema** [comment]
I will merge this PR once the build is successful.

**Comment by vishalraina** [comment]
Thank you @rvema

**Pull request merged**